frontend/spa-assemble-config.json

```diff
@@ -34,7 +34,8 @@
     "@openmrs/esm-patient-vitals-app": "next",
     "@openmrs/esm-primary-navigation-app": "next",
     "@openmrs/esm-service-queues-app": "next",
-    "@openmrs/esm-system-admin-app": "next"
+    "@openmrs/esm-system-admin-app": "next",
+    "@openmrs/esm-help-menu-app": "next"
```

Comment:
List here appears to be organized alphabetically, so I guess we keep that (I had been keeping this list organized by which monorepo they appeared in to make it easier to fill out, but I guess that changed somewhere along the line).

Reply:
Sure will do that.